### ISSUE TITLE
[Console wallet] Wallet recovery panics

### ISSUE THREAD

**Issue by hansieodendaal** [tari-project/tari#5348]
```
Connecting to base node 0000008034cc6453ffae1d0b80...
OK (latency = 534.32ms)
2023-04-21 16:39:58.922004300 +02:00: Recovery process 4% complete (Block 100 of 2493).
2023-04-21 16:39:59.037856200 +02:00: Recovery process 8% complete (Block 200 of 2493).
2023-04-21 16:39:59.208974600 +02:00: Recovery process 12% complete (Block 300 of 2493).
2023-04-21 16:39:59.399291300 +02:00: Recovery process 16% complete (Block 400 of 2493).
2023-04-21 16:39:59.588569500 +02:00: Recovery process 20% complete (Block 500 of 2493).
2023-04-21 16:39:59.761518100 +02:00: Recovery process 24% complete (Block 600 of 2493).
2023-04-21 16:39:59.891141300 +02:00: Recovery process 28% complete (Block 700 of 2493).
2023-04-21 16:40:00.097740300 +02:00: Recovery process 32% complete (Block 800 of 2493).
2023-04-21 16:40:00.211098700 +02:00: Recovery process 36% complete (Block 900 of 2493).
2023-04-21 16:40:00.359149100 +02:00: Recovery process 40% complete (Block 1000 of 2493).
2023-04-21 16:40:00.541011 +02:00: Recovery process 44% complete (Block 1100 of 2493).
2023-04-21 16:40:00.707958 +02:00: Recovery process 48% complete (Block 1200 of 2493).
2023-04-21 16:40:00.861299700 +02:00: Recovery process 52% complete (Block 1300 of 2493).
2023-04-21 16:40:01.021932100 +02:00: Recovery process 56% complete (Block 1400 of 2493).
2023-04-21 16:40:01.188524700 +02:00: Recovery process 60% complete (Block 1500 of 2493).
2023-04-21 16:40:01.317966 +02:00: Recovery process 64% complete (Block 1600 of 2493).
2023-04-21 16:40:01.471944900 +02:00: Recovery process 68% complete (Block 1700 of 2493).
2023-04-21 16:40:01.681136500 +02:00: Recovery process 72% complete (Block 1800 of 2493).
2023-04-21 16:40:01.889205 +02:00: Recovery process 76% complete (Block 1900 of 2493).
2023-04-21 16:40:01.989801700 +02:00: Recovery process 80% complete (Block 2000 of 2493).
2023-04-21 16:40:02.240289 +02:00: Recovery process 84% complete (Block 2100 of 2493).
2023-04-21 16:40:02.355890200 +02:00: Recovery process 88% complete (Block 2200 of 2493).
2023-04-21 16:40:02.457954200 +02:00: Recovery process 92% complete (Block 2300 of 2493).
2023-04-21 16:40:02.716253400 +02:00: Recovery process 96% complete (Block 2400 of 2493).
thread 'tokio-runtime-worker' panicked at 'index out of bounds: the len is 0 but the index is 0', C:\Users\pluto\.tari\Code\@tari-project\base_layer\wallet\src\utxo_scanner_service\utxo_scanner_task.rs:484:37
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by SWvheerden** [comment]
already fixed: #5338

**Comment by hansieodendaal** [comment]
Tested the fix, working perfectly
